[ELEC2313] Add exam January 2019 (#665)
Gp2mv3/Syntheses: src/q7/converto2-ELEC2313/exam/2019/Janvier/All/converto2-ELEC2313-exam-2019-Janvier-All.tex

\documentclass[en]{../../../../../../eplexam}

\hypertitle{Dynamic modelling and control of electromechanical converters}{7}{ELEC}{2313}{2019}{Janvier}{All}
{ELME mécatro 2018-2019}
{Bruno Dehez and Emmanuel de Jaeger}

> \section*{Global proceedings of the exam}

Reviewer: J'ai un peu du mal avec ce mot-là.
```suggestion
\section*{Global course of the exam}
```

The 2 professors ask a question about their part, and it is the one running through their head. There is no paper to draw randomly.

> Some piece of paper are available in order to write the different equations and the schematic blocks.

Reviewer 2: ```suggestion
Some pieces of paper are available in order to write the different equations and the schematic blocks.
```

> They ask for the main equations, and knowing the Concordia, Clarke and Park matrix transformations are an asset.

Reviewer 2: ```suggestion
They ask for the main equations, and knowing the Concordia, Clarke and Park matrix transformations is an asset.
```

    \begin{itemize}
        \item How does it work?
        \item What are the different measures and how do we achieve them?
        (Especially for the current and the position)
        \item How to control the current (if measured or not) and with/without a back emf compensation.
        \item How to correct the error on the position when using a double P controller?
    \end{itemize}
    
\end{enumerate}

\nosolution

\section{Induction machine}

\begin{enumerate}
    \item Explain the scalar control and how to implement it. Explain the flux weakening for this control. 
    \item Scalar control: what do we control and where does the term $\omega_s$ come from? What kinds of sensors do we need for that purpose? 
    \item Give the choice of coordinate system and why. Explain the vector control, its difficulties and give some solutions for these difficulties. Cite the types of sensors needed, and explanation of the flux weakening.
    \item Explain the vector control.
\end{enumerate}

\nosolution

\section{Synchronous machine}
\begin{enumerate}

>     \item Machine with permanent magnets. What does the position of this magnets involve on the value of $L_d$ and $L_q$ (surface mounted magnets, buried magnets and radial magnets)?

Reviewer 2: ```suggestion
    \item Machine with permanent magnets. What does the position of these magnets involve on the value of $L_d$ and $L_q$ (surface mounted magnets, buried magnets and radial magnets)?
```

    \item How to implement a speed control? Detail the procedure in order to implement the vector control for the synchronous machine with permanent magnets.
    \item Vector control of the synchronous machine with surface mounted permanent magnets, what kind of transfer function is implemented in the controllers? Explain the Park transformation, why and how is it used?

>     \item Reluctance machine, what is the expression of the torque? How to control both current? In what frame is this control achieved and how to fix it on the rotor?  When using PWM what problem occurs and how to compensate it?  

Reviewer: ```suggestion
    \item Reluctance machine, what is the expression of the torque? How to control both current? In what frame is this control achieved and how to fix it on the rotor?  When using PWM what problem does occur and how to compensate it?  
```

>     \item Reluctance machine, what is the expression of the torque ? How to control both current ? In what frame is this control achieved and how to fix it on the rotor ?  When using PWM what problem occurs and how to compensate it ?  

Reviewer 2: ```suggestion
    \item Reluctance machine, what is the expression of the torque? How to control both current? In what frame is this control achieved and how to fix it on the rotor?  When using PWM what problem occurs and how to compensate it?  
```
